### ISSUE TITLE
Recycler not accounted for in vessel information

### ISSUE THREAD

**Issue by leomike** [leomike/Kerbalism#2]
The vessel information window does not take into account any recycler present on the ship for the duration of resources, yet the planner in the VAB takes these into account without issues.

**Comment by ShotgunNinja** [comment]
The monitor use the 'delta' amount of resources from the previous simulation step. So it doesn't explicit take into account any module. In fact it doesn't take into account _anything_, made exception for interval-based rules.

If a recycler is present, and is producing/consuming resources, then that will appear in the 'delta' amount and the monitor will consider it in the estimates.

**Comment by leomike** [comment]
Yeah I noticed that. I was thinking of using the same approach as the planner (on mouse over only) to get a better estimate, but haven't had much time to look at that and I have no idea how to Unity works.
